app/BankAccount.php

```diff
@@ -25,46 +25,11 @@ public function tenant(){
     return $this->belongsTo('App\Tenant');
   }
 
-  public static function boot() 
-  {
-    parent::boot();
-
-    self::created(function($model) {
-      if ($model->tenant->payment_details) {
-        $model->tenant->createSubAccount();
-      }
-    });
-
-    self::saving(function($model) {
```

Comment:
looks like we can move all methods exactly how they show up in the models so no need to guess anything moving forward